source/fuzz/fuzzer_pass_duplicate_regions_with_selections.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SPIRV_TOOLS_FUZZER_PASS_DUPLICATE_REGIONS_WITH_SELECTIONS_H
+#define SPIRV_TOOLS_FUZZER_PASS_DUPLICATE_REGIONS_WITH_SELECTIONS_H
+
+#include "source/fuzz/fuzzer_pass.h"
+
+namespace spvtools {
+namespace fuzz {
+
+class FuzzerPassDuplicateRegionsWithSelections : public FuzzerPass {
```

Comment:
I think it would be good to add a comment here (before this line) describing what this fuzzer pass does, as in the other fuzzer passes.